src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Node<T> {
+    private T value;
+    private Node<T> next;
+    private Node<T> prev;
+
+    public Node(Node<T> prev, T value, Node<T> next) {
+        this.value = value;
+        this.next = next;
+        this.prev = prev;
+    }
+
+    public T getValue() {

```

Comment:
[Don't use getters and setters in class Node](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#dont-use-getters-and-setters-in-class-node)